pattern-2/README.md

```diff
@@ -0,0 +1,126 @@
+    3. To deploy Pattern 2, add the following files to the `dist` subdirectory inside the `bosh-deployment` directory.
+
+        * [mssql-jdbc-7.0.0.jre8.jar](https://www.microsoft.com/en-us/download/details.aspx?id=57175)
+        * [mysql-connector-java-5.1.45-bin.jar](https://dev.mysql.com/downloads/connector/j/)
+        * [OpenJDK8U-jdk_x64_linux_hotspot_8u192b12.tar.gz](https://adoptopenjdk.net/archive.html)
+        * [wso2am-2.6.0.zip](https://wso2.com/api-management/install/)
+        * [wso2am-analytics-2.6.0.zip](https://wso2.com/api-management/install/analytics/)
+        * [wso2is-km-5.7.0.zip](https://wso2.com/api-management/install/key-manager/)
+        
+        JDBC Drivers for MySQL and MS SQL are added into the `dist` directory to add flexibility to the deployment. The tile cannot be changed once created, as the tile is immutable. However, options to switch between different databases can be provided.
+        
+        Similarly, Pattern 2 has defined a default key manager packaged with API Manager. However, if a user requires WSO2 Identity Server to be used as the Key Manager, it can be defined in the `pattern-2/tile/tile.yml` file.
+        
+3. Create the BOSH release
+    
+    In order to create the BOSH release, the resource provides two scripts with the deployment. The scripts must be run in the following order:
+    ```bash
+    ./create.sh
+    ```
+    
+    This may take up to 20 minutes. After the build is complete, a bosh-release is completed in the root of the `bosh-deployment` directory.  This release pack is named `wso2am-2.6.0-bosh-release.tgz`.
+    
+## Building the Pivotal Tile for API Manager
+1. Add the [routing-release](https://github.com/cloudfoundry/routing-release/releases/tag/0.178.0) provided by PCF to the root of the `tile` directory. This delivers HTTP and TCP routing for Cloud Foundry.
+2. Copy the bosh release from the `bosh-release` directory into the `tile` directory by issuing the following command.
+    ```bash
+    cd bosh-release
+    mv wso2am-2.6.0-bosh-release.tgz ../tile
+    cd ../tile
+    ```
+3. Run the build script to build the tile by issuing the following command.
+    ```bash
+    ./build.sh
+    ```
+    The build process takes up to 5 minutes. After the build is complete, a tile named `wso2apim-tile-0.0.1.pivotal` is created in the root of the `/tile/product` directory.
+    
+## Install API Manager in PCF
+1. Log in to PCF Ops Manager and upload the tile built by clicking **Import a Product**.
+2. After the tile is uploaded, add the tile to the PCF environment by clicking the + icon next to it.
+    ![import-product](images/import-product.png "Import a Product")
+3. After the tile is added to the environment, click on the **API Manager** tile in the PCF environment to add configurations to the setup.
+    ![API-Manager](images/tile.png "API Manager Tile")
```

Comment:
Shall we fix the following formatting issue?

![image](https://user-images.githubusercontent.com/15902071/53067771-c353de80-34fb-11e9-832e-1430416547fa.png)